suite-native/navigation/src/hooks/useIsKeyboardShown.ts

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import { EmitterSubscription, Keyboard, Platform } from 'react-native';
```

Comment:
I would suggest use of https://kirillzyusko.github.io/react-native-keyboard-controller/docs/api/keyboard-events instead of RN built-in to use one library for everything and be more consistent

Reply:
Fixed in https://github.com/trezor/trezor-suite/pull/16838/commits/443df2cfc901bb01f6a714c7626be6cabf0ed4c2, it's much shorter. 👍